src/handlers/extensions/extensionDeprecated.ts

```diff
@@ -18,14 +21,28 @@ export default async (event: ContractEvent): Promise<void> => {
     colony,
   );
 
-  await mutate<
-    UpdateColonyExtensionByColonyAndHashMutation,
-    UpdateColonyExtensionByColonyAndHashMutationVariables
-  >(UpdateColonyExtensionByColonyAndHashDocument, {
-    input: {
-      colonyId: colony,
-      hash: extensionHash,
-      isDeprecated: deprecated,
-    },
-  });
+  const { data } =
+    (await query<
+      GetColonyExtensionByHashAndColonyQuery,
+      GetColonyExtensionByHashAndColonyQueryVariables
+    >(GetColonyExtensionByHashAndColonyDocument, {
+      extensionHash,
+      colonyAddress: colony,
+    })) ?? {};
+
+  const extensionId = data?.listColonyExtensions?.items[0]?.id;
+
+  if (extensionId) {
+    await mutate<
+      UpdateColonyExtensionByAddressMutation,
+      UpdateColonyExtensionByAddressMutationVariables
+    >(UpdateColonyExtensionByAddressDocument, {
+      input: {
+        id: extensionId,
+        colonyId: colony,
```

Comment:
Once you have the extension id you don't need to pass `colonyId` or `hash` to the mutation